### PR TITLE
Altinn Authorization

### PR DESCRIPTION
**Pull request by Ceredron**
## Description
We need to integrate with Altinn Authorization to allow resource/service owners to use other Altinn tools to manage access. It is also a pre-requisite to integrate with existing Platform services like Altinn Events.

## Related Issue(s)
- #310 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

**Comment by Ceredron** [comment]
Awaiting a deployment by team authorixation (in dev, not tt02 yet)

File: Test/Altinn.Broker.Tests/FileControllerTests.cs [hunk 1]
```diff
@@ -335,6 +323,18 @@ public async Task UploadFile_ChecksumSetWhenInitialized_SameChecksumSetAfterUplo
         Assert.Equal(checksum, fileDetails.Checksum);
     }
 
+    [Fact]
+    public async Task SendFile_UsingUnregisteredUser_Fails()
+    {
+        var file = FileInitializeExtTestFactory.BasicFile();
```
Comment (RagnarFatland-Avanade) [inline Test/Altinn.Broker.Tests/FileControllerTests.cs:329]: Consider formatting for readabiliity, splitting up in // Arrange, // Act, // Assert

File: src/Altinn.Broker.Integrations/Altinn/Authorization/XacmlMappers.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    /// </summary>
+    internal const string ActionId = "a";
+
+    /// <summary>
+    /// Resource id for multi requests. Inde should be appended.
+    /// </summary>
+    internal const string ResourceId = "r";
+
+    /// <summary>
+    /// Creates an action category with the porovided action type as an attribute
```
Comment (RagnarFatland-Avanade) [inline src/Altinn.Broker.Integrations/Altinn/Authorization/XacmlMappers.cs:49]: porovided => provided

File: Test/Altinn.Broker.Tests/FileControllerTests.cs (hunk 1 above)
Comment (Ceredron) [reply Test/Altinn.Broker.Tests/FileControllerTests.cs:330]: Fixed

**Review by RagnarFatland-Avanade — APPROVED** [review]
Looks good so far :)

**Pull request merged**